plonky2/src/gates/random_access.rs

```diff
@@ -24,9 +24,15 @@ use crate::plonk::vars::{
 /// A gate for checking that a particular element of a list matches a given value.
 #[derive(Copy, Clone, Debug)]
 pub struct RandomAccessGate<F: RichField + Extendable<D>, const D: usize> {
+    // Number of bits in the index (log2 of the list size).
```

Comment:
nit: IMO best to use `///` for such comments so that tools will associate them with the field.

Reply:
Ahh right, thanks!